blog/tag/seo.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: tag
+tag: seo
+permalink: /blog/tag/seo/
+redirect_from:
```

Comment:
@chrisarusso I'll refrain from referencing that, as it will add unnecessary noise to the article.